### PR TITLE
Build vs buy blog post

### PR DESCRIPTION
**Pull request by st0nebraker**
Adds a blogpost & along with image asset, closing #5564

### Changelog
- Adds new blog post `.md` along with the associated asset

### Test

1. Ensure prettier has standardized the proposed changes.
2. Navigate to `/blog/things-to-know-before-building-a-code-search-tool` and ensure blog post renders as expected
3. Test responsive sizing
4. Navigate to `/blog` index & ensure new blog card renders as expected

### CODE REVIEW

File: content/blogposts/2022/things-to-know-before-building-a-code-search-tool.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: '3 things to know before building a custom, in-house code search tool '
+authors:
+  - name: Quinn Slack
+    url: https://twitter.com/sqs
+publishDate: 2022-07-11T10:00-07:00
```
Comment (st0nebraker) [inline content/blogposts/2022/things-to-know-before-building-a-code-search-tool.md:6]: I'll update this with the actual date we decide to publish this 😄

File: content/blogposts/2022/things-to-know-before-building-a-code-search-tool.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: '3 things to know before building a custom, in-house code search tool '
```
Comment (bretthayes) [inline content/blogposts/2022/things-to-know-before-building-a-code-search-tool.md:2]:
```suggestion
title: '3 things to know before building a custom, in-house code search tool'
```

File: content/blogposts/2022/things-to-know-before-building-a-code-search-tool.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+authors:
+  - name: Quinn Slack
+    url: https://twitter.com/sqs
+publishDate: 2022-07-11T10:00-07:00
+description: ' As your company’s codebase grows, devs will start needing code search to understand, fix, and automate across the codebase. Here are a few questions you should ask yourself before deciding to build code search inside your company.'
```
Comment (bretthayes) [inline content/blogposts/2022/things-to-know-before-building-a-code-search-tool.md:7]:
```suggestion
description: 'As your company’s codebase grows, devs will start needing code search to understand, fix, and automate across the codebase. Here are a few questions you should ask yourself before deciding to build code search inside your company.'
```

**Review by bretthayes — CHANGES_REQUESTED** [review]
Looks good to me! Just two minor things:

File: content/blogposts/2022/things-to-know-before-building-a-code-search-tool.md (hunk 1 above)
Comment (bretthayes) [reply content/blogposts/2022/things-to-know-before-building-a-code-search-tool.md:6]: Let's deploy this today

**Review by bretthayes — APPROVED** [review]

**Pull request merged**